jenkins/kola/packet.sh

```diff
@@ -27,6 +27,29 @@ else
   CPIO_URL="http://${BASE_PATH}/flatcar_production_pxe_image.cpio.gz"
 fi
 
+if [[ "${BOARD}" == "amd64-usr" ]]; then
```

Comment:
```suggestion
if [[ "${KOLA_TESTS}" == "" ]]; then
  KOLA_TESTS="*"
fi

# cl.basic includes cl.internet which is run on multiple instance types
cl_basic_included="$(bin/kola list --filter "${KOLA_TESTS}" | grep cl.basic)"
if [[ "${BOARD}" == "amd64-usr" ]] && [[ "${cl_basic_included}" != ""  ]]; then
```
Should we maybe skip this if there's a filter set for kola which does not include cl.basic?

Reply:
Done, also added the platform option for the filter (default is qemu)